vspace/src/main/java/edu/asu/diging/vspace/core/aspects/ExhibitionDataAspect.java

```diff
@@ -42,4 +50,22 @@ public void setExhibition(JoinPoint jp) {
             }
         }
     }
+    
+    @Around("execution(public * edu.asu.diging.vspace.web..ExhibitionSpaceController.*(..))")
```

Comment:
you want this to happen for all controllers for the public side, don't you? not just the exhibitionspacecontroller.